packages/windows/postinstall.ps1

```diff
@@ -73,4 +73,23 @@ if (-not (Get-Service -Name $serviceName -ErrorAction SilentlyContinue)) {
     Write-Host "Service $serviceName already installed."
 }
 
+# Register "wazuh-agent" as an Event Log source if not already present
+$logName = "Application"
+$sourceName = "wazuh-agent"
+
+$eventLogExists = [System.Diagnostics.EventLog]::SourceExists($sourceName)
+
+if (-not $eventLogExists) {
+    try {
+        Write-Host "Registering event log source: $sourceName"
+        New-EventLog -LogName $logName -Source $sourceName
+        Write-Host "Event log source registered successfully."
+    } catch {
+        Write-Host "Failed to register event log source: $_"
+        exit 1
+    }
+} else {
+    Write-Host "Event log source '$sourceName' is already registered."
```

Comment:
can this be an issue?

Reply:
No, if the name wazuh-agent is already registered in Eventlog there is no need to register it again, the messages will be displayed correctly anyway.